packages/shared-ui/src/elements/canvas-controller/canvas-controller.ts

```diff
@@ -210,6 +214,12 @@ export class CanvasController extends SignalWatcher(LitElement) {
     } else {
       this.sideNavItem = sideNavItem;
     }
+
+    window.addEventListener("bbflowgen:generate:complete", () => {
```

Comment:
This is a bit of a hack, we probably should use the actual to reset the state rather than side-load it via an event listener like that.